### DISCUSSION TITLE
Show volume levels

### DISCUSSION THREAD

**Discussion by pcar75** [jamulussoftware/jamulus#3294] [UI Related]
Bonjour,
I would find it very helpful if the volume ( 0-100 % , 100 being level at start of red bars) level would be displayed in a track; it could be static or dynamic when mouse-hovering over the track.  I 'travel' to different servers and it would be nice to jot down the levels of tracks as I'm hearing them. The levels that are saved only happen upon program exit.

Thanks for your consideration.

**Comment by ann0see** [comment]
Hi. What do you mean by tracks?
The actual levels shown in the GUI are very course by design to avoid transmitting too much metadata, thus you wouldn't gain much.

**Reply by pcar75** [reply]
I meant wherever there is a fader (mixing side of console). It's easier to remember a number than to guess or count at what bar-graph level a fader is. 
I'm not sure I understand your reasoning .  If I modify a fader level , the information still has to be sent to the server for it to adjust and send my whole mix back ; no need to send back a confirmation of what I sent it. !?!  In any case, information (i.e metadata) still is sent ...

**Reply by ann0see** [reply]
Ok. I thought you meant the volume level getting back.  So you want to get e.g. the LED number as level?

**Comment by dcorson-ticino-com** [comment]
Jamulus always assumes that you are listening to the sound. 
As such there is no real need to know any numbers.  
If you hear if everything is well balanced it is good, and probably much better than trying to do this numerically.

**Reply by pcar75** [reply]
>  Jamulus always assumes that you are listening to the sound. 

In the sense of not listening to direct monitoring , absolutely.
In the sense of initial session setup, this is sufficient for open jams as there are known and unknown participants who join. 
However in the sense of bands with regular or core participants on public or private servers sessions, it would be more helpful to swiftly calibrate setup with numbers.

But a better solution could be to enhance the 'group' feature, to maintain initial setup ...

**Reply by ann0see** [reply]
There's the auto adjust all faders setting. Maybe have a look at that

**Reply by pcar75** [reply]
Oh ! I had not seen this (new ?) one  !   Thanks. 
My experience is that it seems to take a snapshot ( an instant, not a short duration ) of levels and adjust each according to some level ( -3dBFS ?) . If an unmuted channel is silent it will set it to the level specified in 'New client level'.  
Soooo ... best use the auto-adjust with every participants simultaneously  output their loudest sound ?!

**Reply by ann0see** [reply]
It isn't new. See https://github.com/jamulussoftware/jamulus/issues/1071

**Reply by pcar75** [reply]
Thanks ! Very interesting discussion !
I will need to test it to understand it's behavior in real situation.
